db/models.py

```diff
@@ -49,4 +58,62 @@ class MovieSession(models.Model):
     )
 
     def __str__(self) -> str:
-        return f"{self.movie.title} {str(self.show_time)}"
+        return (f"{self.movie.title} "
+                f"{self.show_time.strftime('%Y-%m-%d %H:%M:%S')}")
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession,
+                                      on_delete=models.CASCADE,
+                                      related_name="session_tickets")
```

Comment:
just tickets would be fine :)

Reply:
`    movie_session = models.ForeignKey(MovieSession,
                                      on_delete=models.CASCADE,
                                      related_name="session_tickets")
    order = models.ForeignKey(Order,
                              on_delete=models.CASCADE,
                              related_name="order_tickets")`
                              
i can`t name it tickets, cause i need to have diff between this 2 backward relation, how i can distinguish if they will have same names?

Reply:
Simple because they will exist in different models. One will be MovieSession.tickets another Order.tickets